rllib/core/learner/learner.py

```diff
@@ -82,6 +82,13 @@
 POLICY_LOSS_KEY = "policy_loss"
 VF_LOSS_KEY = "vf_loss"
 ENTROPY_KEY = "entropy"
+LOGPS_KEY = "logps"
```

Comment:
Could we leave these specific constants in SACLearner?

Don't want to clutter the base Learner with any algo-specific stuff.

Reply:
Yes we can pull them over - I guess we need these constants at least in the DQN and CQL again.